ВП-3

ptsmor opened the pull request:
Бонусный проект фильтр друзей

ptsmor committed:
ВП-3

Бонусный проект фильтр друзей

AlexandrAkimov commented on projects/filter-friends/css/styles.css:
```diff
@@ -0,0 +1,69 @@
+.friend { display: flex; align-items: center; padding: .4rem 0; cursor: grab; }
+.friend:hover { background-color: #f0f0f0; }
+.friend__photo { margin: 0 .4rem; width: 2.3rem; min-width: 2.3rem; height: 2.3rem; border-radius: 50%; pointer-events: none; }
+.friend__content { font-size: .6rem; font-weight: 500; color: #505050; }
```
можно переносить стили на другие строчки)) читать так легче)

AlexandrAkimov commented on projects/filter-friends/index.js:
```diff
@@ -0,0 +1,84 @@
+import Element from './elements';
+import DnD from './dnd';
+
+Model.login(51423341, 2)
```
такие данные лучше вынеси в отдельные переменные, и экспортируй эти переменные из другого файла. а файл помести в гит игнор. чтобы в гитхаб не попал твой айди с ВК. Так-то ниче страшного, но на будущее для практики сделай. в будущем понадобится 100%. когда будешь передавать какие-нибудь пароли от БД. правда обычно для этого используются .env  - переменные окружения

ptsmor committed:
changes VP-3

- перенес стили
- создал в корне сборки файл .env (для себя) поместил туда код приложения)
